docs/cugraph/source/index.rst

```diff
@@ -1,15 +1,40 @@
-Welcome to RAPIDS Graph documentation
-=====================================
-RAPIDS Graph covers a range of graph libraries and packages, that includes:
-
-* cugraph: GPU-accelerated graph algorithms
-* cugraph-ops: GPU-accelerated GNN aggregators and operators
-* cugraph-service: multi-user, remote GPU-accelerated graph algorithm service
-* cugraph-pyg:  GPU-accelerated extensions for use with the PyG framework
-* cugraph-dgl:  GPU-accelerated extensions for use with the DGL framework
-* wholegraph: shared memory-based GPU-accelerated GNN training
-
-cuGraph is a library of graph algorithms that seamlessly integrates into the RAPIDS data science ecosystem and allows the data scientist to easily call graph algorithms using data stored in GPU DataFrames, NetworkX Graphs, or even CuPy or SciPy sparse Matrices.
+RAPIDS Graph documentation
+==========================
+.. image:: images/cugraph_logo_2.png
+   :width: 600
+
+*Making graph analytics fast and easy regardless of scale*
+
+
+.. list-table:: RAPIDS Graph covers a range of graph libraries and packages, that includes:
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Core
+     - GNNß
+     - Extension
+   * - :abbr:`cugraph (Heavy-weight Python wrapper with lot of guard rails)`
```

Comment:
minor typo, but the bigger feedback is that IMO "heavy-weight" has bad connotations, do we want to use that?
```suggestion
   * - :abbr:`cugraph (Python wrapper with lots of guard rails)`
```

Reply:
fixed